mixin/lib/thanos-grafana-builder/builder.libsonnet

```diff
@@ -78,6 +78,24 @@ local utils = import '../utils.libsonnet';
     ],
   },
 
+  qpsErrTotalPerLabelPanel(selectorErr, selectorTotal, dimensions, perLabel):: {
```

Comment:
Leaving a decision documented: I created this `qpsErrTotalPerLabelPanel` function because in many places `qpsErrTotalPanel` is used for generic error charts (i.e. in the Compactor and Ruler dashboards).

As future work we could investigate whether those widgets have some sort of code/reason/message field that is worth surfacing in the widget's legend.